test/widgets/docs_screenshots/ss_clock.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2021 elParaguayo
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+import pytest
+
+from libqtile.widget import Clock
+
+
+@pytest.fixture
+def widget():
+    yield Clock
+
+
+@pytest.mark.parametrize("screenshot_manager", [{}, {"format": "%d/%m/%y %H:%M"}], indirect=True)
+def ss_clock(screenshot_manager):
+    screenshot_manager.take_screenshot()
```

Comment:
At its simplest a screenshot file is:

1. A `widget` fixture returning the widget class (can be monkeypatched as necessary)
2. A `ss_` function which is parametrized with the various configs.
3. A call to `screenshot_manager.take_screenshot()`